app/views/derivatives/_derivatives.html.erb

```diff
@@ -0,0 +1,21 @@
+<hr class="small" id="sources" />
+
+<h3 class="mb-1/4">Sources</h2>
+
+<p><em><small>This code uses parts of or entire other codes. This could be because they built off of them, used parts of them, used them for inspiration, or other reasons.</small></em></p>
```

Comment:
I think "This post utilizes other codes either in part or in full." reads better. Also see below for some general wordsmithing.